tests/conftest.py

```diff
@@ -770,10 +771,25 @@ def keep(self):
         self.keep_shared = True
         return {"env": self.env, "libs": self.libs, "project": self.project_dir}
 
-    def setup_for_snippet(self, snippet, autostd=True):
+    def setup_for_snippet(self, snippet, autostd=True, install_v2_modules: List[LocalPackagePath] = []) -> None:
+        """
+        :param install_v2_modules: Indicates which V2 modules should be installed in the compiler venv
+        """
         self.setup_for_snippet_external(snippet)
-        Project.set(Project(self.project_dir, autostd=autostd))
-        loader.unload_inmanta_plugins()
+        project = Project(self.project_dir, autostd=autostd)
+        Project.set(project)
+        project.use_virtual_env()
```

Comment:
Shouldn't we make sure to deactivate this as a cleanup step?

Reply:
That is handled in the `deactive_venv` fixture. This is an autouse fixture

Reply:
That's interesting, I didn't know of that one. I've written basically that in another fixture yesterday. Guess I can get rid of the cleanup section.